src/ZPublisher/HTTPResponse.py

```diff
@@ -465,44 +492,38 @@ def setBody(self, body, title='', is_error=0,
             try:
                 body = bytes(body)
             except UnicodeError:
```

Comment:
If it is `BytesIO`, then it won't raise `UnicodeError` (I was asking "is this exception reachable?").